template/package.json

```diff
@@ -24,6 +24,7 @@
     "@react-navigation/native-stack": "^6.9.1",
     "@reduxjs/toolkit": "^1.8.6",
     "i18n-js": "^4.1.1",
+    "axios": "^0.27.2",
```

Comment:
Why we use this version instead of the latest version

Reply:
I downgrade the version to 0.27.2 to fix this issue
https://github.com/saigon-technology/rn-base-project/pull/10

Reply:
done

Reply:
I still see the "^" in "^0.27.2". Please check again

Reply:
done